ipywidgets/widgets/widget_selection.py

```diff
@@ -406,8 +564,17 @@ class SelectionSlider(_SelectionNonempty):
         help="Update the value of the widget as the user is holding the slider.").tag(sync=True)
 
 @register
+@_doc_subst
 class SelectionRangeSlider(_MultipleSelectionNonempty):
-    """Slider to select a single item from a list or dictionary."""
+    """
+    Slider to select multiple contiguous items from a list.
```

Comment:
Can you add a note that the index, value, and label attributes indicate the first and last items in the range, rather than every element in the range?